packages/sdk/src/types/runtime/user.ts

```diff
@@ -50,6 +68,13 @@ export class User implements UserLike {
         if (from.id !== undefined) this._id = from.id;
         if (from.name !== undefined) this._name = from.name;
         if (from.properties !== undefined) this._properties = from.properties;
+        if (from.groups !== undefined) {
+            if (typeof from.groups === 'number') {
+                this._groups = UserGroup.FromPacked(this._context, from.groups);
+            } else {
+                this._groups = from.groups;

```

Comment:
I don't understand this bit - is this because we allow to store this both as a class and a number here?